rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -8,6 +8,9 @@
 #include "rsx_decode.h"
 #include "Emu/Cell/PPUCallback.h"
 
+#include <fstream>
```

Comment:
Ok. By the way the draft spec for C++17 has been published and it provides a std::filesystem implementation which is based on boost filesystem. Do you think it could be useful to replace fs::file and related class with the one from boost and switch to the official one when standard libraries are updated ?